types/underscore/index.d.ts

```diff
@@ -3533,39 +3545,44 @@ declare module _ {
         ): Extract<keyof V, string> | undefined;
 
         /**
-        * Return a copy of the object, filtered to only have values for the whitelisted keys
-        * (or array of valid keys).
-        * @param object Object to strip unwanted key/value pairs.
-        * @keys The key/value pairs to keep on `object`.
-        * @return Copy of `object` with only the `keys` properties.
-        **/
-        pick<T, K extends keyof T>(obj: T, ...keys: K[]): Pick<T, K>;
-        pick<T, K extends keyof T>(obj: T, keys: K[]): Pick<T, K>;
-        pick<T, K extends keyof T>(obj: T, predicate: ObjectIterator<T[K], boolean>): Pick<T, K>;
+         * Return a copy of `object` that is filtered to only have values for
+         * the whitelisted keys (or array of keys).
+         * @param object The object to pick specific keys in.
+         * @param keys The keys to keep on `object`.
+         * @returns A copy of `object` with only the `keys` properties.
+         **/
+        pick<V, K extends string>(object: V, ...keys: K[]): _Pick<V, K>;
```

Comment:
`_.pick` and `_.omit` apply a single level of flattening to the rest arguments, so:

```suggestion
        pick<V, K extends string>(object: V, ...keys: (K | K[])[]): _Pick<V, K>;
```

Reply:
Good call, that's definitely an improvement worth making. I've really got to get better at actually looking at the source for functions I'm trying to write definitions for rather than just documentation + old definitions to catch nuances like these.